src/cascadia/TerminalSettingsEditor/MainPage.xaml

```diff
@@ -9,13 +9,43 @@
       xmlns:local="using:Microsoft.Terminal.Settings.Editor"
       xmlns:mc="http://schemas.openxmlformats.org/markup-compatibility/2006"
       xmlns:muxc="using:Microsoft.UI.Xaml.Controls"
-      Background="{ThemeResource SettingsPageBackground}"
       mc:Ignorable="d">
 
     <Page.Resources>
         <ResourceDictionary>
             <ResourceDictionary.MergedDictionaries>
                 <ResourceDictionary Source="CommonResources.xaml" />
+
+                <ResourceDictionary>
+                    <ResourceDictionary.ThemeDictionaries>
+
+                        <!--
+                            These two values are
+                            "SolidBackgroundFillColorTertiary". We can't just put
+                            those here though as StaticResources, because XAML will
+                            evaluate their values when the App is first loaded, and
+                            we'll always use the value from the OS theme, regarless
+                            of the requested theme. Kinda the same thing we've had
+                            to do with TabViewBackground in the past.
+                        -->
+
+                        <ResourceDictionary x:Key="Dark">
+                            <Color x:Key="SettingsPageBackground">#282828</Color>

```

Comment:
I'm cool with this as a quick fix but we might want to throw a BODGY comment in here. We'll need to fix this if we ever support a different version of the OS lol